README.md

```diff
@@ -60,18 +60,21 @@ out when using this package.
     ```go
     import (
         _ "crypto/sha256"
-   	    _ "crypto/sha512"
+        _ "crypto/sha512"
     )
     ```
     This may seem inconvenient but it allows you replace the hash 
     implementations with others, such as https://github.com/stevvooe/resumable.
  
-2. Even though `digest.Digest` may be assemable as a string, _always_ 
+2. Even though `digest.Digest` may be assemblable as a string, _always_ 
     verify your input with `digest.Parse` or use `Digest.Validate`
     when accepting untrusted input. While there are measures to 
     avoid common problems, this will ensure you have valid digests
     in the rest of your application.
 
+3. While alternative encodings of hash values (digests) are possible (for
+    example, base64), this package deals exclusively with hex-encoded digests.
```

Comment:
As far as I can tell, with d1caf2031 the package position will be:

1. The algorithm identifier (e.g. ‘sha256’) identifies both the hash algorithm (e.g. [SHA-256][1]) and the encoding (e.g. [lowercase base16][2].
2. Algorithm identifiers which use encodings other than lowercase base 16 are allowed.
3. This package will reject pull requests which would allow support for algorithm identifiers which use encodings other than lowercase base 16.
4. This package will reject pull requests which would allow library consumers to register their own externally-defined `Algorithm` unless that externally-defined `Algorithm` uses a lowercase base 16 encoding.

Putting that all together, that's “you *can* define a new algorithm identifier using an encoding other than lowercase base 16, but if you do you won't be able to use go-digest at all, and we're not interested in providing a library that will work with you”.  I find that position unfortunate, when the alternatives (either requiring hex encoding for all algorithm identifiers, or providing a framework that would support algorithm identifiers which used other encodings) seem so easy to adopt.  But maybe I'm misunderstanding something about the package position?

[1]: https://tools.ietf.org/html/rfc4634
[2]: https://tools.ietf.org/html/rfc4648#section-8